tests/test_website.py

```
@@ -31,101 +47,260 @@
             d = d[k]
         return d
     except KeyError:
-        # If a key is not in MOCK_CONFIG_DATA, return the default.
-        # This is important if the code being tested calls config.get for other keys.
+        # Fallback to default if key not in MOCK_CONFIG_DATA
+        # This is important if the code being tested calls config.get for other keys
+        # not mocked explicitly (e.g. general.date_format if not in MOCK_CONFIG_DATA)
+        if key_tuple == ('general', 'date_format'): return '%Y-%m-%d'
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)